colaraz/lms/static/js/discussion/colaraz_discussion_utils.js

```diff
@@ -0,0 +1,53 @@
+$(document).on('click', 'button.action-vote', function(event) {
+
+    if ($(this).closest('article.discussion-article').length <= 0) {
+      return;
+    }
+
+    let isVotingDisabled = $(this).closest('li.actions-item').hasClass('is-disabled');
+
+    if (isVotingDisabled) {
+      console.log('Voting is disabled. An author can\'t vote his own post or comment.');
```

Comment:
```suggestion
      console.error('Voting is disabled. An author can\'t vote on his own post or comment.');
```

Reply:
Actually, it's not an error. It's edX own functionality/ feature whatever you say. This logging message is just to make the user know that you can't vote as an author.